core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -157,15 +159,24 @@ public void onReady() {}
   public abstract void request(int numMessages);
 
   /**
+   * Equivalent as {@code cancel(null, null)}.
+   */
+  public final void cancel() {
```

Comment:
This is not API Stable.  Anyone who implemented their own ClientCall will now break.